fe/fe-core/src/main/java/com/starrocks/http/rest/ShowMetaInfoAction.java

```diff
@@ -93,21 +95,30 @@ public static void registerAction(ActionController controller) throws IllegalArg
     @Override
     public void execute(BaseRequest request, BaseResponse response) {
         String action = request.getSingleParameter("action");
+        // check param empty
+        if (Strings.isNullOrEmpty(action)) {
+            response.appendContent("Missing parameter");
+            writeResponse(request, response, HttpResponseStatus.BAD_REQUEST);
+            return;
+        }
         Gson gson = new Gson();
-        response.setContentType("application/json");
-
         switch (Action.getAction(action.toUpperCase())) {
             case SHOW_DB_SIZE:
+                response.setContentType("application/json");
                 response.getContent().append(gson.toJson(getDataSize(true)));
                 break;
             case SHOW_FULL_DB_SIZE:
+                response.setContentType("application/json");
                 response.getContent().append(gson.toJson(getDataSize(false)));
                 break;
             case SHOW_HA:
+                response.setContentType("application/json");
                 response.getContent().append(gson.toJson(getHaInfo()));
                 break;
             default:
-                break;
+                response.appendContent("Invalid parameter: " + action);
```

Comment:
I guess you can just clear the contentType in this branch, instead of adding them one by one in every other branches.

Reply:
updated as suggested